xarray/core/variable.py

```diff
@@ -633,11 +633,18 @@ def to_index(self) -> pd.Index:
         """Convert this variable to a pandas.Index"""
         return self.to_index_variable().to_index()
 
-    def to_dict(self, data: bool = True, encoding: bool = False) -> dict:
+    def to_dict(
+        self, data: bool | str = "list", encoding: bool = False
+    ) -> dict[str, Any]:
         """Dictionary representation of variable."""
-        item = {"dims": self.dims, "attrs": decode_numpy_dict_values(self.attrs)}
-        if data:
-            item["data"] = ensure_us_time_resolution(self.values).tolist()
+        item: dict[str, Any] = {
+            "dims": self.dims,
+            "attrs": decode_numpy_dict_values(self.attrs),
+        }
+        if data is not False:
+            item["data"] = ensure_us_time_resolution(self.to_numpy())
```

Comment:
```suggestion
            item["data"] = ensure_us_time_resolution(self.data)
```

This will preserve the underlying array type and not compute by default.

Reply:
this was a wee bit more complicated... to get from dask to list